groovy/postInit/chemistry/inorganic_chemistry/elements/s_block/group2/BerylliumChain.groovy

```diff
@@ -308,6 +300,121 @@ DISTILLERY.recipeBuilder()
         .duration(100)
         .buildAndRegister()
 
-//BERTRANDITE CHAIN (SPECIALIZED)
+ROASTER.recipeBuilder()
+        .inputs(ore('dustBeryllium'))
+        .fluidInputs(fluid('oxygen') * 1000)
+        .outputs(metaitem('dustBerylliumOxide') * 2)
+        .EUt(Globals.voltAmps[2])
+        .duration(100)
+        .buildAndRegister()
 
+//BERTRANDITE CHAIN (SPECIALIZED)
 //HIGH PURITY CHAIN
+BR.recipeBuilder()
+        .inputs(ore('dustBertrandite') * 1)
+        .fluidInputs(fluid('sulfuric_acid') * 4100) //100 for Ammonium Alum
+        .outputs(metaitem('dustSiliconDioxide') * 6)
+        .fluidOutputs(fluid('impure_bertrandite_leach') * 1000) //Be4Si2O7(OH)2 + 4H2SO4 -> 4BeSO4 + 2SiO2 + 5H2O
+        .EUt(Globals.voltAmps[2])
+        .duration(100)
+        .buildAndRegister()
+
+BR.recipeBuilder()
+        .inputs(ore('dustAmmoniumSulfate') * 3) // 1/5 mol
```

Comment:
if you add 100 sulfuric acid per bertrandite, you would only be dissolving 0.0667 moles of Al impurity, you need to increase the sulfuric acid input to 4150, which would give the following recipe.

BR.recipeBuilder()
.inputs(ore('dustAmmoniumSulfate') * 3)
.fluidInputs(fluid('impure_bertrandite_leach') * 10000)
.chancedOutput(metaitem('dustAmmoniumAlum') * 4, 8000, 0)
.fluidOutputs(fluid('bertrandite_leach') * 2000)